test/fixtures/fixtures.js

```diff
@@ -1035,7 +1035,7 @@ export function getTargetingKeys() {
     ],
     [
       'foobar',
-      ['0x0', '300x250', '300x600']
+      ['300x250', '300x600', '0x0']
```

Comment:
Why does this need to change?

Reply:
Im assuming its because I am gathering the custom keys immediately and applying them to the bids with matching ad unit codes instead of the previous logic. It used to take in all bids received and cycle through them in a reducer, and then as it cycled, it would cycle through the accumulator bids and update them if they found custom keys.

Reply:
OK this makes sense - it's an aggregate from multiple bids, and it looks like the previous order was random, so this should be fine.